connmanager/incoming_connections.go

```diff
@@ -7,13 +7,17 @@ func (c *ConnectionManager) checkIncomingConnections(connSet connectionSet) {
 		return
 	}
 
+	numConnectionsOverMax := len(connSet) - c.maxIncoming
```

Comment:
then->than. And maybe rename connSet to incomingConnectionSet so it'll be clearer that all of the left connections are incoming connections.